verification/curator-service/ui/src/components/BulkCaseForm.tsx

```diff
@@ -250,6 +265,7 @@ class BulkCaseForm extends React.Component<
         const { classes } = this.props;
         return (
             <Formik
+                validationSchema={BulkFormSchema}
```

Comment:
You may also want to include `validateOnChange={false}` That way the form validation only kicks in on blur and form submission. It's a nicer UX (doesn't start showing error messages as you are typing) and also greatly decreased lag on the CaseForm.

Reply:
Good call, I'll add.